src/Data/Scripts/DataSQL.sql

```diff
@@ -0,0 +1,51 @@
+CREATE TABLE IF NOT EXISTS category (
+    id UUID PRIMARY KEY,
+    rol VARCHAR NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS localization (
+    id UUID PRIMARY KEY,
+    name varchar(100),
+    geom GEOMETRY(Point, 4326)
+);
+
+CREATE TABLE IF NOT EXISTS client (
+    id UUID PRIMARY KEY,
+    localization_id UUID NOT NULL,
+    name VARCHAR NOT NULL,
+    emailAddress VARCHAR NOT NULL,
+    categoryId UUID NOT NULL
+);
```

Comment:
The work schedule column is missing, it is necessary to know the worker's availability.